src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = {};
+
+  sourceString.split(';').forEach((declaration) => {
+    let [property, value] = declaration.split(':');
+
+    if (property && value) {
+      property = property.trim();
+      value = value.trim();
+      styles[property] = value;
+    }
+  });
```

Comment:
[CODE KNOWLEDGE] - Don't use iteration methods to modify an already existing array/object. In this case, you are directly modifying the `styles` object inside `forEach`. Instead, consider using `reduce` to create a new object with the desired properties.